CONTRIBUTING.md

```diff
@@ -1,22 +1,20 @@
 ## 🤗 Welcome
 
-:+1::tada: First off, We are really glad you're reading this, because we need volunteer developers to help improve the Field Mapping Tasking Manager (FMTM)! :tada::+1:
-
-We welcome and encourage contributors of all skill levels, and we are committed to making sure your participation is inclusive, enjoyable, and rewarding. If you have never contributed to an open source project before, we are a good place to start, and we will make sure you are supported every step of the way. If you have **any** questions, please ask!
+👍🎉 First off, We are really glad you're reading this, because we need volunteer developers to help improve the Field Mapping Tasking Manager (FMTM)! Welcome to Field Mapping Tasking Manager (FMTM)! We're excited to have you join our community of contributors. Whether you're new to open source or an experienced developer, we value your contributions and are here to support you every step of the way. If you have any questions or need assistance, please don't hesitate to ask. 🎉👍
 
 You can see an overview of the project and the process we have gone through in developing FMTM so far in these [slides](https://docs.google.com/presentation/d/1UrBG1X4MXwVd8Ps498FDlAYvesIailjjPPJfR_B4SUs/edit#slide=id.g15c1f409958_0_0) .
 
-Furthermore, there are many ways to contribute to the **Field Mapping Tasking Manager (FMTM)**, which includes:
+How to Contribute
+
+There are several ways to contribute to the Field Mapping Tasking Manager:
 
 ## Testing
```

Comment:
As these headings go below `How to contribute` now, they can be level 3 headings `###`